### PR TITLE
Namespace notification - email+irc schema 

### PR DESCRIPTION
**Pull request by urdaalex**
schemas copied from
- http://schemas.taskcluster.net/notify/v1/irc-request.json
- http://schemas.taskcluster.net/notify/v1/email-request.json

### CODE REVIEW

File: schemas/namespace-request.yml [hunk 1]
```diff
@@ -7,16 +7,10 @@ properties:
...
-    properties:
-        method:
-            type:           string
-            title:          Contact Method
-            description:    "The contact method (eg. irc, email)"
-        id:
-            type:           string
-            title:          The contact id
-            description:    "The contact id (eg. username, email address)"
+    description:    The contact Information
```
Comment (djmitche) [inline schemas/namespace-request.yml]: It may be useful to indicate in the description that these will be handed off to the notification service.

Comment (urdaalex) [inline schemas/namespace-request.yml]: Yep, I agree. The changes have been made.

**Comment by Hexcles** [comment]
Hi @urdaalex , I cleaned up the merge a bit to make the history of this branch cleaner (otherwise Github seemed to have trouble figuring out the diff base correctly). You'll need to `git pull --force` at your local `namesapce-notification_v2` branch. Sorry for any trouble.

Some more information if you're curious: if you took a look at `tig`, the previous history looked like this:
history1

In case of this kind of relatively "messy" history, usually you can try `git rebase master` (in your feature branch) and look at the history again. As a comparison, it's now:
history2

Finally, you'll need to `git push --force` (which is fine in this case, i.e. to a working/PR branch) since the history is modified.

**Comment by urdaalex** [comment]
Thanks @Hexcles, that's something i need to start doing for sure, I appreciate the extra work you've done :)

**Pull request merged**